overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/ignition-ostree-transposefs.sh

```diff
@@ -139,13 +138,15 @@ should_autosave_rootfs() {
         return
     fi
     local agcount
-    eval $(xfs_info "${root_part}" | grep -o 'agcount=[0-9]*')
+    xfsinfo=$(xfs_info "${root_part}")
+    agcount=$(echo "$xfsinfo" | grep -o 'agcount=[0-9]*' | tr -dc '0-9')
```

Comment:
might be easier to read if we use `cut` here:

```suggestion
    agcount=$(echo "$xfsinfo" | grep -o 'agcount=[0-9]*' | cut -d '=' -f 2)
```

Reply:
The use of `eval` here that was present before is IMO a giant red flag...I can't say I understand why he chose to do it, but explicitly setting the variable is way better!